lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/interfaces.cpp

```diff
@@ -8,40 +8,40 @@ StepperInterface::StepperInterface(uint8_t PWM1_P, uint8_t PWM2_P,
     : s_(steps, DIR1_P, DIR2_P), PWM1_P_{PWM1_P}, PWM2_P_{PWM2_P},
       DIR1_P_{DIR1_P}, DIR2_P_{DIR2_P}, en_{0}, steps_{steps}, speed_{speed},
       step_size_{step_size} {
-    pinMode(PWM1_P_, OUTPUT);
-    pinMode(PWM2_P_, OUTPUT);
-    s_.setSpeed(speed_);
+  pinMode(PWM1_P_, OUTPUT);
+  pinMode(PWM2_P_, OUTPUT);
+  s_.setSpeed(speed_);
 }
 
 void StepperInterface::step(StepperDir dir) {
-    if (en_) {
-        s_.step(step_size_ * dir);
-    }
+  if (en_) {
```

Comment:
What is en_? Give a more descriptive name

Reply:
should've just wrote `enabled_`, smh at past me lol

Reply:
done